packages/opentelemetry-cloud-monitoring-exporter/src/utils.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { TimeSeries } from './types';
+
+/** Returns an array with arrays of the given size. */
+export function partitionList(list: TimeSeries[], chunkSize: number) {
```

Comment:
It would be better to copy the array first, since this will mutate the original array passed into the function. E.g. 
` const listCopy === [...list]`;

Reply:
Thanks for the feedback, updated!